### PR TITLE
Added CentOS-Stream to distros

### PR DESCRIPTION
**Pull request by k0ste**
Containers added https://github.com/netdata/helper-images/pull/219

### CODE REVIEW

**Comment by Ferroin** [comment]
On a side note, CI failures are likely due to packaging issues with the new systemd-journal plugin, in which case this will need a rebase after https://github.com/netdata/netdata/pull/15733 is merged to resolve those issues.

**Comment by k0ste** [comment]
Thank you, please also merge https://github.com/netdata/helper-images/pull/222

**Comment by k0ste** [comment]
Core switched to Community, rebased. Seems error's only for DEB - unrelated

**Review by tkatsoulas — DISMISSED** [review]

**Comment by k0ste** [comment]
Merge it? Wanna to continue work on the #14551

File: .github/scripts/pkg-test.sh [hunk 1]
```diff
@@ -14,7 +14,8 @@ install_debian_like() {
 
   # Install Netdata
   # Strange quoting is required here so that glob matching works.
-  apt-get install -y $(find /netdata/artifacts -type f -name 'netdata*.deb' ! -name '*dbgsym*' ! -name '*cups*' ! -name '*freeipmi*') || exit 3
+  apt-get install -y "$(find /netdata/artifacts -type f -name 'netdata*.deb' \
+! -name '*dbgsym*' ! -name '*cups*' ! -name '*freeipmi*')" || exit 3
```
Comment (ilyam8) [inline .github/scripts/pkg-test.sh:18]:
```suggestion
  apt-get install -y $(find /netdata/artifacts -type f -name 'netdata*.deb' \
! -name '*dbgsym*' ! -name '*cups*' ! -name '*freeipmi*') || exit 3
```

Comment (ilyam8) [reply .github/scripts/pkg-test.sh:18]: >  Seems error's only for DEB - unrelated

It totally related, you made a change to `install_debian_like` in `pkg-test.sh`. Remove `"`.

**Review by Ferroin — APPROVED** [review]

File: .github/scripts/pkg-test.sh (hunk 1 above)
Comment (k0ste) [reply .github/scripts/pkg-test.sh]: Disabled this shellcheck

**Review by Dim-P — APPROVED** [review]

**Comment by k0ste** [comment]
Please merge

**Pull request merged**